pkg/cmd/cli/describe/printer.go

```diff
@@ -36,7 +36,7 @@ var (
 	projectColumns          = []string{"NAME", "DISPLAY NAME", "STATUS"}
 	routeColumns            = []string{"NAME", "HOST/PORT", "PATH", "SERVICE", "LABELS", "INSECURE POLICY", "TLS TERMINATION"}
 	deploymentColumns       = []string{"NAME", "STATUS", "CAUSE"}
-	deploymentConfigColumns = []string{"NAME", "TRIGGERS", "LATEST"}
+	deploymentConfigColumns = []string{"NAME", "REVISION", "REPLICAS", "TRIGGERED BY"}
```

Comment:
Do we have current on status now?

On Wed, Feb 3, 2016 at 8:00 PM, Michail Kargakis <notifications@github.com>
wrote:

> In pkg/cmd/cli/describe/printer.go
> <https://github.com/openshift/origin/pull/7010#discussion_r51814973>:
>
> > @@ -36,7 +36,7 @@ var (
> >  	projectColumns          = []string{"NAME", "DISPLAY NAME", "STATUS"}
> >  	routeColumns            = []string{"NAME", "HOST/PORT", "PATH", "SERVICE", "LABELS", "INSECURE POLICY", "TLS TERMINATION"}
> >  	deploymentColumns       = []string{"NAME", "STATUS", "CAUSE"}
> > -	deploymentConfigColumns = []string{"NAME", "TRIGGERS", "LATEST"}
> > +	deploymentConfigColumns = []string{"NAME", "REVISION", "REPLICAS", "TRIGGERED BY"}
>
> Also not showing replicas so far has been really disturbing. We need
> though to show current alongside desired, showing only desired is
> misleading. We also need to make use of -o wide and put images, containers,
> and the selector under it. Preparing a pull for that.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/7010/files#r51814973>.
>